ctapipe_io_nectarcam/__init__.py

```diff
@@ -334,6 +365,65 @@ def fill_nectarcam_event_container_from_zfile(self, event):
         # Unpack FEB counters and trigger pattern
         self.unpack_feb_data(event)
 
+    def fill_trigger_info(self, array_event):
+        tel_id = self._tel_id
+
+        nectarcam = array_event.nectarcam.tel[tel_id]
+        tib_available = nectarcam.evt.extdevices_presence & 1
+        ucts_available = nectarcam.evt.extdevices_presence & 2
+
+        # fill trigger time using UCTS timestamp
+        trigger = array_event.trigger
+        trigger.time = nectarcam.evt.ucts_timestamp
```

Comment:
This needs to be an `astropy.time.Time`, `nectarcam.evt.ucts_timestamp` should be an int in nanoseconds, correct?

You can use: https://github.com/cta-observatory/ctapipe_io_lst/blob/e5e8939a8bae4dd88409181ac714f8b2bea9a86b/ctapipe_io_lst/event_time.py#L92

Reply:
Thanks, fixed